roles/nova-data/tasks/libvirt.yml

```diff
@@ -14,7 +14,7 @@
     - qemu-kvm={{ nova.qemu_kvm_version }}
     - "{{ nova.qemu_system_package }}={{ nova.qemu_kvm_version }}"
     - qemu-system-common={{ nova.qemu_kvm_version }}
-    - librbd1={{ nova.librdb1_version }}
+    - librbd1={{ ceph.client_version[os] }}
```

Comment:
oh, there could be error if we specify nova.librbd1_version different with ceph.client_version.
Thinking that we have ceph enabled, and specified ceph.client_version = 10.2.2, this will make librbd1=10.2.2 installed. If we set nova.librbd1_version=10.2.3 in all.yml, this may cause package conflicts.